openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -281,6 +282,36 @@ def linkedin_add_to_profile_url(self):
                 self.overview.display_name, user_certificate.mode, cert_url, certificate=user_certificate,
             )
 
+    @property
+    def related_programs(self):
+        """
+        Returns related program data if the effective_user is enrolled.
+        Note: related programs can be None depending on the course.
+        """
+        if self.effective_user.is_anonymous:
+            return
+
+        meter = ProgramProgressMeter(self.request.site, self.effective_user)
+        inverted_programs = meter.invert_programs()
+        related_programs = inverted_programs.get(str(self.course_key))
+
+        if related_programs is None:
+            return None
+
+        related_progress = meter.progress(programs=related_programs)
+        progress_by_program = {
+            progress.get("uuid"): progress for progress in related_progress
```

Comment:
Here and below: use single quotes instead of double quotes. It's our convention at edX